zulipterminal/model.py

```diff
@@ -305,6 +306,21 @@ def get_messages(self, *,
             return True
         return False
 
+    def get_topics_in_stream(self, stream_list: Iterable[int]) -> bool:
```

Comment:
The commit that adds this method states it fetches topics at startup, but that commit doesn't fetch them all at startup - that's in another commit.

Reply:
Agreed. I'll amend the commit title to something like 
`Fetch all topics with specified streams_ids'